packages/story-editor/src/components/floatingMenu/constants.js

```diff
@@ -28,6 +28,11 @@ export const SELECTED_ELEMENT_TYPES = {
   NONE: 'none',
 };
 
+export const TOOLBAR_POSITIONS = {
+  element: 'fixed_to_element',
+  top: 'fixed_to_top',
+};
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick:** I think we use uppercase for nested const variables more often than not, albeit we are not consistent 😉 

![image](https://user-images.githubusercontent.com/21036224/184891251-41fb4e9b-bc0b-4eae-905f-b62c8f86c804.png)